xdiag/basis/electron_distributed/apply/apply_terms.hpp

```diff
@@ -11,6 +11,7 @@
 #include <xdiag/basis/electron_distributed/apply/apply_raise_lower.hpp>
 #include <xdiag/basis/electron_distributed/apply/apply_szsz.hpp>
 #include <xdiag/basis/electron_distributed/apply/apply_u.hpp>
+#include <xdiag/basis/apply_identity_distributed.hpp>
```

Comment:
please follow the suggestions iin apply_identity_distributed.hpp. Move to apply_identity.hpp